src/svc/nifid/schemas/SubscriptionStatusArray.yml

```diff
@@ -0,0 +1,5 @@
+properties:
+  subscriptions:
+    type: array
+    items:
+      $ref: "./SubscriptionStatus.yml"
```

Comment:
we'll need to extend this definition as it currently just has a property of isActive